### DISCUSSION TITLE
Elección dataset (cantidad de filas)

### DISCUSSION THREAD

**Discussion by yohelperez** [rramosp/ai4eng.v1#242] [00 - General]
Hola Profe,

A nuestro equipo le interesa el dataset de esta [competición ](https://www.kaggle.com/competitions/tmdb-box-office-prediction/data), en la que se quiere predecir las ganancias en taquilla que tendria una pelicula, cumple con todas las condiciones sin embargo tenemos una duda en cuanto a la cantidad de filas:
El archivo de test tiene 4399 filas, sample_submission tiene 4399 filas y el de train 3001. No estoy seguro si este sea valido ya que en el requisito de las filas dice que debe tener 5000 pero no se si es la suma de todos los archivos o cada uno de estos tres debe tener 5000.
Sería entonces este dataset valido?

**Comment by rramosp** [comment]
veo que el dataset de entrenamiento tiene 23 columnas ... al final os
quedàis cortos de ambos columnas y filas (en el de train) ... mejor que
escojáis otro datset

On Wed, Mar 8, 2023 at 2:18 PM Yohel Perez ***@***.***> wrote:

> Hola Profe,
>
> A nuestro equipo le interesa el dataset de esta competición
> <https://www.kaggle.com/competitions/tmdb-box-office-prediction/data>, en
> la que se quiere predecir las ganancias en taquilla que tendria una
> pelicula, cumple con todas las condiciones sin embargo tenemos una duda en
> cuanto a la cantidad de filas:
> El archivo de test tiene 4399 filas, sample_submission tiene 4399 filas y
> el de train 3001. No estoy seguro si este sea valido ya que en el requisito
> de las filas dice que debe tener 5000 pero no se si es la suma de todos los
> archivos o cada uno de estos tres debe tener 5000.
> Sería entonces este dataset valido?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/242>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIIQ4XBVJ557UF6DHETW3DLPHANCNFSM6AAAAAAVUE7YAU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by yohelperez** [reply]
Gracias Profe,

Encontré esta [competición](https://www.kaggle.com/competitions/allstate-claims-severity/data). El reto es crear un modelo que prediga la probabilidad de que un conductor inicie un reclamo de seguro de automóvil en el próximo año

El archivo train tiene 116 columnas categoricas y 14 continuas al igual que el archivo test. Sin embargo la cantidad de filas son 188318 para el train y 125547 para el test. 

Recomendaría tomarlo acotar la cantidad de filas que se toman a 5000?

**Reply by imEag** [reply]
Entre mayor número de columnas puedes tener un mejor desempeño en el modelo pero requiere mayor esfuerzo computacional (lo que resulta en mayor tiempo de entrenamiento), lo ideal es que encuentres el punto medio que veas adecuado.
